config/routes.rb

```diff
@@ -19,4 +19,7 @@
   resources :users do
     get 'dashboard', on: :member
   end
+
+  #Define route for Create Event Button
+  get "/createEvent" , to: "application#createEvent" , as: "createEvent"
```

Comment:
Please use snake case consistently for methods.
``..., as: "create_event"``
So you can redirect to the action with a ``create_event_path()`` method and not ``createEvent_path()``